src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -186,8 +265,9 @@ private static void checkJSONFilesAreWritten(final String workspace) {
 	Assert.assertTrue(new File(workspace, GenomicsDBConstants.DEFAULT_VCFHEADER_FILE_NAME).exists());
     }
 
-    private static void checkGenomicsDBAgainstExpected(final String workspace, final SimpleInterval interval, final String expectedCombinedVCF) throws IOException {
+    private static void checkGenomicsDBAgainstExpected(final String workspace, final SimpleInterval interval, final String expectedCombinedVCF, final String referenceFile, final boolean testAll) throws IOException {
         final GenomicsDBFeatureReader<VariantContext, PositionalBufferedStream> genomicsDBFeatureReader =
+                getGenomicsDBFeatureReader(workspace, referenceFile, !testAll);
 		getGenomicsDBFeatureReader(workspace, b38_reference_20_21);
```

Comment:
I think this is a spurious line from the old code.

Reply:
removed